manifests/integreatly-cloud-resources/0.26.0/cloud-resource-operator.clusterserviceversion.yaml

```diff
@@ -328,4 +328,5 @@ spec:
   maturity: alpha
   provider:
     name: Integreatly
+  replaces: cloud-resources.v0.24.1
```

Comment:
@laurafitzgerald  I think this need to replace 0.25.0 instead, 0.26.0 CSV should be skipping 0.25.0 and replacing 0.25.0 - ideally, the CSV (although not used at all as we are using index and bundle, it would be good to keep consistent with what's in the bundle)

Reply:
These csv's are only here for documentation purposes, they have no effect on the logic since we brought in bundles. will make the change to include v0.25.0